test/es6.js

```diff
@@ -320,6 +320,36 @@ describe('import', function() {
     });
 });
 
+describe.skip('dynamic import', function() {
```

Comment:
Couldn’t you get these tests to work by putting in object literals with the ast instead ofblocking on espree?

Reply:
I hadn't tried that. I'll give it a go